### ISSUE TITLE
Feature Request: Use mouse to attract or repel particles

### ISSUE THREAD

**Issue by chexbox** [fnky/particle-life#3]
Sometimes I sit watching and wish I could change it a tiny bit. 
I think left click should attract and right click repel or vise versa.

**Comment by fnky** [comment]
Absolutely, that has been something I thought of adding but never got around to do it.
